.travis.yml

```diff
@@ -24,27 +10,12 @@ env:
     - FULLSTACK=0 UITESTS=1
     - GH_PUBLISH=true FULLSTACK=1
     - SCHEDULER_FULLSTACK=1
-matrix:
-  include:
-    - perl: 5.26
-      env: FULLSTACK=0 UITESTS=0
-before_install:
-  - cpanm local::lib
-  - eval "$(perl -Mlocal::lib=${HOME}/perl_modules)"
-  - mkdir -p $HOME/chrome
-  - if ! test -f $HOME/chrome/chromedriver; then curl -s https://chromedriver.storage.googleapis.com/2.36/chromedriver_linux64.zip | funzip - > $HOME/chrome/chromedriver; fi
-  - chmod a+x $HOME/chrome/chromedriver
-  - export PATH=$PATH:$HOME/chrome
-install:
-  - gem install sass
-  - cpanm -nq --installdeps --with-feature=coverage .
+before_install: yes
+install: yes
+before_script:
+  - docker pull dasantiago/openqa-tests  
```

Comment:
How we will update the image later? I guess it's better to setup dockerhub or an organization for openqa to build this image and change to it before merge